portfolio.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="portfolio.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.1/css/all.css"
+    />
+    <title>My Portfolio</title>
+  </head>
+
+  <body>
+    <header>
+      <nav class="topnav">
+        <a class="mylogo" href="#mylogo">My Logo</a>
+        <ul class="nav__links">
+          <li class="line"><a href="#portfolio"></a></li>
+          <li class="line"><a href="#about"></a></li>
+          <li class="line"><a href="#contact"></a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="backimage">
+      <div class="welcome">
+        <h2>I'm Amrendra Glad to see you!</h2>
+      </div>
+      <div class="dev-details">
+        <p>
+          I’m a software developer! skilled in React JS, Express JS, Javascript,
+          CSS, HTML, JAVA, Ruby, Ruby on Rails. Hands-on experience in
+          implementing Web Development technologies. Interested in developing
+          softwares. I can help you build a product , feature or website Look
+          through some of my work and experience! If you like what you see and
+          have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="connect-icon">
+        <div class="connect">LET'S CONNECT</div>
+        <div class="social-icons">
+          <a
+            href="https://twitter.com/amrendrak_"
+            target="_blank"
+            rel="noopener noreferrer"
+            ><i class="fab fa-twitter"></i
+          ></a>
+          <a
+            href="https://www.linkedin.com/in/amrendraakumar/"
+            target="_blank"
+            rel="noopener noreferrer"
+            ><i class="fab fa-linkedin-in"></i
+          ></a>
+          <a
+            href="https://github.com/amrendrakind"
+            target="_blank"
+            rel="noopener noreferrer"
+            ><i class="fab fa-github"></i
+          ></a>
+        </div>
+      </div>
```

Comment:
Don't forget to make this change as well 😁 Due to semantic reasons, it's considered a better approach to wrap a list of icons in a ul tag to group them! Each of the `a` tags should be wrapped in a `li` tag as well. Please introduce these changes in your project so that we meet the project requirements 😎